xml/deployment_installing_nodes.xml

```diff
@@ -387,6 +386,122 @@
+   NTP configuration. No manual NTP configuration is required on the cluster
+   nodes, they synchronize time with the admininstration node.
+  </para>
+
+  <sect2>
+   <title><command>caasp-admin-setup</command> Details</title>
+   <para>
+    The general purpose of <command>caasp-admin-setup</command> is to collect
+    all information needed to successfully start the &productname; containers.
+   </para>
+   <para>
+    When <command>caasp-admin-setup</command> is executed it determines which
+    cluster node image to use according to the cloud framework. For this
+    operation to succeed outgoing traffic on port <literal>443</literal> to the
+    Internet must be permitted. The code will access the <literal>Public Cloud
+    Information Tracker</literal> service operated by SUSE. This service
+    provides information about all images ever released to the Public Cloud by
+    SUSE. The latest available cluster node image for this version of
+    &productname; will be used. This initial outreach and image filtering
+    introduces a small startup delay before the command line options are
+    processed or the wizard mode starts.
+   </para>
+  </sect2>
+  <sect2>
+   <title>Providing SSL Certificate and Key</title>
+   <para>
+    You may choose to supply your own SSL certificate and key for initial
+    access the dashboard, with the <literal>--ssl-crt</literal> and
+    <literal>--ssl-key</literal> options or by answering the question
+    <quote>Would you like to use your own certificate from a known (public
+    or self signed) Certificate Authority?</quote> with
+    <literal>y</literal>.
+   </para>
+   <para>
+    In order to use your own SSL certificate and key you must upload the files
+    to the admin node into a location of your choice. This location is then
+    provided to the setup code. For example, if your certificate is called
+    <filename>my-velum.crt</filename> and you uploaded it to
+    <filename>/tmp</filename> then the <command>caasp-admin-setup</command>
+    code expects <filename>/tmp/my-velum.crt</filename> as the location for
+    the SSL certificate. The same concept applies to the SSL key. The
+    certificate and key will be placed in the appropriate locations on the
+    admin node.
+   </para>
+  </sect2>
+  <sect2>
+   <title>Velum Administrator Credentials</title>
+   <para>
+    Velum is the name of the administrative dashboard web interface. The setup
+    code will ask for an e-mail address and a password if not supplied with
+    the <literal>--admin-email</literal> and
+    <literal>--admin-password</literal> arguments. These are the
+    administrative credentials to log into the Velum dashboard. The e-mail
+    used does not have to be an e-mail associated with your &scc; account.
+    Please do not forget the values you enter, as they cannot be recovered.
+   </para>
+  </sect2>
+  <sect2>
+   <title>Registering with &scc;</title>
```

Comment:
This is duplicated again in `deployment_configuration.xml` section `sec.configuration.suseconnect`

Add a statement saying you can always register this later and link to the other section.

Reply:
This is not really a duplicate as here we describe a step in the workflow with caasp-admin-setup with unique parameters. However, we can reduce the text and linking to the other section for later registration is useful.